### DISCUSSION TITLE
Communication using BLE

### DISCUSSION THREAD

**Discussion by syedadil19** [pschatzmann/ESP32-A2DP#501] [Q&A] [answered]
Hi Phil, I was trying the "communications-ble-server-send.ino" example and was able to upload your code to my ESP32 to test it out. But, how did you confirm the transmission of the audio? I was using the nRF Connect for Mobile App and was only able to view the output as a hex string. Mind sharing your method with me?

**Comment by pschatzmann** [comment, marked as answer]
I used the LightBlue app to test the connection, but above all the the corresponding receiver Arduino sketch to play back the transmitted audio
